apps/jobs/views.py

```diff
@@ -696,6 +697,31 @@ def remove_documents(self, request, pk=None):
             f"{document_type.capitalize()} removed successfully", status.HTTP_200_OK
         )
 
+    @action(detail=False, methods=["post"])
+    def find_user(self, request):
+        """
+        to retrive user as per the filters in the post body.
+        """
+
+        data = request.data
+        qualification = data.get("qualification", None)
+        experience = data.get("experience", None)
+        location = data.get("location", None)
+
+        queryset = User.objects.all()
+
+        if qualification:
+            queryset = queryset.filter(qualification__icontains=qualification)
+
+        if experience:
+            queryset = queryset.filter(experience=experience)
+
+        if location:
+            queryset = queryset.filter(address__icontains=location)
```

Comment:
needs refactoring,

 if experience is not None:
            queryset = queryset.filter(experience=experience)
            
 **is not None** check for the experience filter, as it could be 0, and a falsy check (if experience:) would not work correctly in that case